### ISSUE TITLE
Feature request: Limit speed

### ISSUE THREAD

**Issue by absywabsy** [ipsingh06/seedsync#135]
When seedsync downloads something it literally takes my 100% of my speed meaning I cannot browse while the download is happening.

Is there a way to limit it to 90-95%?

**Comment by quadcom** [comment]
I don't believe there is but you could reduce the number of threads. The whole purpose of Seedsync is to DL as fast as possible which other sync clients are not able to do. If you prefer a slower sync I would recommend syncthing.

**Comment by absywabsy** [comment]
Ah right, the reason I use SeedSync over Syncthing is because my seedbox does not include LFTP as part of it's monthly traffic limit, but it would include it for Syncthing.
